### DISCUSSION TITLE
Migrating from 3.5.0 to 3.6.0

### DISCUSSION THREAD

**Discussion by jonsonasMafjiozas** [osm-search/Nominatim#2113] [Q&A]
I'm currently running on 3.5.0 with the automatic updates running.
When migrating from 3.5.0 to 3.6.0 should i stop the automatic updates first and then follow the steps in the migration page?
or can i do it while the updates are running?

**Comment by mtmail** [comment]
You should stop updates. After the migration you can start them again, they will continue where they left of.

**Reply by jonsonasMafjiozas** [reply]
Thanks!
